### PR TITLE
Small tweaks, requesting review

### PR DESCRIPTION
**Pull request by hbarovertwo**
Requesting review on this readme please, @JoelPasvolsky 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ If no puzzle file is specified, `puzzle_files/example1.txt` is used by default.
 There are three constraints used in formulating this problem:
 1. Minimize the difference between squares of left-hand side and right-hand side of equation (minimum of zero).
```
Comment (JoelPasvolsky) [inline README.md:34]: How is bullet 1 a constraint? As described here, it looks like an objective.

File: README.md [hunk 2]
```diff
@@ -33,12 +33,12 @@ If no puzzle file is specified, `puzzle_files/example1.txt` is used by default.
...
 2. Assignments of digits to letters must be unique, thus a puzzle may contain no more than 10 unique letters.
-3. The first letter must be non-zero, unless the puzzle contains one letter long components.
+3. The first letters must be non-zero, unless the puzzle contains one letter long components.
```
Comment (JoelPasvolsky) [inline README.md:36]: Do you mean "The first letter of each word must be assigned a non-zero digit"?
Also "contains one-letter-long components" or better yet "contains one-letter words"

File: README.md [hunk 3]
```diff
@@ -33,12 +33,12 @@ If no puzzle file is specified, `puzzle_files/example1.txt` is used by default.
...
 
 # Code Specifics
 Puzzle files containing larger words are generally harder to solve.
 Scaling biases can help the solver arrive at an optimal solution for larger problems with large energy scales.
-Due to the probabilistic nature of the solver, it may take several runs of a puzzle file to find a solution.
+Due to the probabilistic nature of the solver, it may take several runs with a puzzle file to find a solution.
```
Comment (JoelPasvolsky) [inline README.md:41]: How about something like this? 
"Puzzles become harder to solve for increasing word length. For large problems---which have high energy scales---scaling up the biases can help the solver arrive at an optimal solution. Due to the probabilistic nature of the solver, you may have to run the puzzle a few times to find a solution."

**Review by JoelPasvolsky — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hbarovertwo) [reply README.md:34]: You are right, it is an objective

File: README.md (hunk 3 above)
Comment (hbarovertwo) [reply README.md:41]: I guess I should be explicit in which direction I'm scaling - so down.

File: README.md (hunk 1 above)
Comment (hbarovertwo) [reply README.md]: Updated the wording and distinguished objective/constraints.

**Pull request merged**